pycbc/waveform/parameters.py

```diff
@@ -547,7 +555,8 @@ def docstr(self, prefix='', include_label=True):
 # passed to the waveform generators in lalsimulation, but are instead applied
 # after a waveform is generated. Distance, however, is a parameter used by
 # the waveform generators.
-location_params = ParameterList([tc, ra, dec, polarization])
+location_params = ParameterList([tc, ra, dec, polarization,
```

Comment:
I'll accept this here, but we may need to be caereful with breaking the LIGO code if merging this into the main branch.